app/src/main/java/com/github/se/studentconnect/ui/eventcreation/CreatePrivateEventScreen.kt

```
@@ -48,19 +51,33 @@ object CreatePrivateEventScreenTestTags {
 @Composable
 fun CreatePrivateEventScreen(
     modifier: Modifier = Modifier,
+    navController: NavHostController? = null,
```

`navController` should not be nullable. This screen needs navigation when saving finishes. Make it a required parameter instead of nullable, otherwise a missing controller will cause a silent failure.